services/121-service/src/payments/fsp-integration/safaricom/repositories/safaricom-transfer.repository.ts

```diff
@@ -17,25 +15,15 @@ export class SafaricomTransferRepository extends Repository<SafaricomTransferEnt
     );
   }
 
-  public async createAndSaveSafaricomTransferData(
-    safaricomDoTransferResult: DoTransferReturnType,
-    transaction: TransactionEntity,
-  ): Promise<any> {
+  public async storeSafaricomTransfer(
```

Comment:
Why encapsulate the save function in this extra function, instead of just creating the Entity object and calling save in the calling function, like we "always do"?

Reply:
Updated!